Zend/zend_portability.h

```diff
@@ -531,12 +531,12 @@ static zend_always_inline double _zend_get_nan(void) /* {{{ */
 # define ZEND_INTRIN_HAVE_IFUNC_TARGET 1
 #endif
 
-#if (defined(__i386__) || defined(__x86_64__))
+#if (defined(__i386__) || defined(__x86_64__)) && !defined(__OpenBSD__)
 # if PHP_HAVE_SSSE3_INSTRUCTIONS && defined(HAVE_TMMINTRIN_H)
 # define PHP_HAVE_SSSE3
 # endif
 
-# if PHP_HAVE_SSE4_2_INSTRUCTIONS && defined(HAVE_NMMINTRIN_H)
+# if PHP_HAVE_SSE4_2_INSTRUCTIONS && defined(HAVE_NMMINTRIN_H) && !defined(__OpenBSD__)
```

Comment:
This code is nested inside the outer `!defined(__OpenBSD__)` check, so it's not necessary to check if here again. Same for the next check.

Reply:
Ah true :)